mobile/build.gradle

```diff
@@ -51,6 +51,12 @@ android {
     }
 }
 
+configurations {
+    compile.exclude module: 'stax'
+    compile.exclude module: 'stax-api'
+    compile.exclude module: 'xpp3'
+}
```

Comment:
Why this?

Reply:
Otherwise the build fails with:
````
processing "javax/xml/stream/EventFilter.class":
Ill-advised or mistaken usage of a core class (java.* or javax.*)
when not building a core library.
This is often due to inadvertently including a core library file
in your application's project, when using an IDE (such as
Eclipse). If you are sure you're not intentionally defining a
core class, then this is the most likely explanation of what's
going on.
````

see https://travis-ci.org/openhab/openhab-android/builds/334803598#L3119